### PR TITLE
Homework 16 02 20 remastered

### PR DESCRIPTION
**Pull request by cofano88**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 * {
   box-sizing: border-box;
-  margin-block-start: 0;
-  margin-block-end: 0;
+  /* margin-block-start: 0;
```
Comment (tpotapchik) [inline assets/css/style.css]: Remove it if you don't need it

File: assets/css/style.css [hunk 2]
```diff
@@ -27,31 +27,32 @@
 }
 
 header {
-  background-image: url(../img/header.jpg);
+  background-image: linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0.4),
+      rgba(0, 0, 0, 0.4)
+    ),
+    url(../img/header.jpg);
```
Comment (tpotapchik) [inline assets/css/style.css:34]: Add no-repeat

File: assets/css/style.css [hunk 3]
```diff
@@ -27,31 +27,32 @@
...
+      rgba(0, 0, 0, 0.4),
+      rgba(0, 0, 0, 0.4)
+    ),
+    url(../img/header.jpg);
   background-position: center;
-}
-
-.filter-header {
-  background-color: rgba(0, 0, 0, 0.4);
+  background-repeat: no-repeat;
```
Comment (tpotapchik) [inline assets/css/style.css:36]: Seems you don't need itnow

File: assets/css/style.css [hunk 4]
```diff
@@ -27,31 +27,32 @@
...
 }
 
 nav {
-  display: inline-block;
-  position: absolute;
-  top: 32px;
-  left: 850px;
+  float: left;
+  margin-top: 25px;
+  margin-left: 565px;
```
Comment (tpotapchik) [inline assets/css/style.css]: suspicious value

File: assets/css/style.css [hunk 5]
```diff
@@ -209,6 +219,8 @@ nav {
   color: #000000;
   font-family: "Open Sans", sans-serif;
   font-weight: 700;
+  margin-top: 0;
```
Comment (tpotapchik) [inline assets/css/style.css]: If you don't want to use origin margin value you can define it at the top

*{margin:0;}

File: assets/css/style.css [hunk 6]
```diff
@@ -491,15 +519,17 @@ ul.features li {
...
-  background-image: url(../img/custom.jpg);
-  background-position: center;
-  height: 300px;
-}
+  background-image: linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0.4),
+      rgba(0, 0, 0, 0.4)
+    ),
+    url(../img/custom.jpg);
```
Comment (tpotapchik) [inline assets/css/style.css:499]: Add no-repeat

File: index.html [hunk 7]
```diff
@@ -210,14 +208,14 @@ <h3 class="name">Jeremy H.</h3>
             </div>
           </div>
 
-          <div class="review">
+          <div class="review clearfix">
```
Comment (tpotapchik) [inline index.html]: Why clearfix here?

**Review by tpotapchik — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (cofano88) [reply index.html]: Because it have 3 float elements

Comment (cofano88) [reply index.html]: sorry, just found it) My fail.

**Pull request merged**